### PR TITLE
Add flappybit.py example

### PR DESCRIPTION
**Pull request by funkyHat**
from November 2015 LdnPyDojo

### CODE REVIEW

File: examples/flappybit.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        for y in range(top):
+            display.set_pixel(column, y, 4)
+            if column == 0 and bird == y:
+                lost_status = True
+        for y in range(bottom):
+            display.set_pixel(column, 4-y, 4)
+            if column == 0 and bird == (4-y):
+                lost_status = True
+    if lost_status:
+        display.print(Image.SAD)
```
Comment (deshipu) [inline examples/flappybit.py]: We renamed this to display.show() now.

**Comment by tomviner** [comment]
Wow, it's hard but I managed to go round the loop a couple of times!

**Comment by funkyHat** [comment]
Should I redo this pull request as a single commit (after figuring out how to do that :)) or is it OK as is?

**Comment by dpgeorge** [comment]
Please squash into a single commit (git rebase -i master).  Please also PEP8-ify it :)

File: examples/flappybit.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    sleep(100)
+    if -256 < accelerometer.get_y() < 450:
+        bird = max(0, bird - 1)
+    elif 568 < accelerometer.get_y() < 1024:
+        bird = min(4, bird + 1)
+    
+    display.clear()
+    display.set_pixel(0, bird, 9)
+    
+    pos_terrain = pos//terrain_multiplier
```
Comment (dpgeorge) [inline examples/flappybit.py]: pep8

File: examples/flappybit.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    lost_status = False
+    for column, (top, bottom) in enumerate(terrain[pos_terrain:pos_terrain+5]):
```
Comment (dpgeorge) [inline examples/flappybit.py]: pep8 with +5

File: examples/flappybit.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        for y in range(top):
+            display.set_pixel(column, y, 4)
+            if column == 0 and bird == y:
+                lost_status = True
+        for y in range(bottom):
+            display.set_pixel(column, 4-y, 4)
```
Comment (dpgeorge) [inline examples/flappybit.py]: pep8 with 4-y

**Comment by deshipu** [comment]
In order to squash it, do this:

    git checkout master
    git pull
    git checkout your-branch  
    git rebase -i master
    # rreplace all the "pick" but first with "squash"
    git push --force

**Comment by funkyHat** [comment]
Here's my next go at this :) Any other changes needed?

(out of interest, which PEP8 checker are you using, pep8 didn't pick up the spacing between infix operators in function calls as being a violation?)

**Comment by willingc** [comment]
@funkyHat Looks good. Nice job on the rebase and squash too. Small typo in the comment block

**Comment by dpgeorge** [comment]
> out of interest, which PEP8 checker are you using

I just use my eyes :)  So maybe I got it wrong.

**Comment by funkyHat** [comment]
Whoops. Typo corrected!

**Comment by dpgeorge** [comment]
Music is now a separate module, so you need to do `import music` instead of `from microbit import music`.  @funkyHat Can you please fix this and then rebase against latest master?

**Comment by funkyHat** [comment]
Done

**Comment by dpgeorge** [comment]
Great, thanks!

**Pull request merged**